hatchet.json

```diff
@@ -1 +1,8 @@
-{}
+{
+  "heroku": [
+    "heroku/node-js-getting-started"
+  ],
+  "rack": [
```

Comment:
Might want to change this to be a different directory. Something more generic like "nodejs" maybe. In Ruby `rack` is a type of application "rack app".